js/ad_manager_vast.js

```diff
@@ -282,9 +283,8 @@ OO.Ads.manager(function(_, $) {
      * @returns {boolean} Returns true if the root tag is valid otherwise it returns false.
      */
     this.isValidRootTagName = function(vastXML) {
-      var rootTagName = (vastXML && vastXML.firstChild) ? vastXML.firstChild.tagName || '' : '';
-      if (rootTagName.toUpperCase() != "VAST") {
-        OO.log("VAST: Invalid VAST XML for Tag Name: " + rootTagName);
+      if ($(vastXML).find("VAST").length === 0) {
+        OO.log("VAST: Invalid VAST XML");
```

Comment:
Hmm, it's not possible for there to be another VAST tag anywhere else in a VAST 3.0 response (as specified by the VAST 3 specs) so if there is another one, then it would be an error in the XML.